modules/project-structure/01-project-structure.qmd

```diff
@@ -0,0 +1,180 @@
+---
+title: Project structure
+subtitle: 'In this topic, you will learn about the project structure and how to manage your project in R properly.'
+title-block-banner: yes
+engine: knitr
+filters:
+  - webr-teachr
+execute: 
+  echo: false
+---
+
+```{r}
+knitr::opts_chunk$set(
+    fig.width = 8,
+    fig.height = 8,
+    fig.align = "center")
+```
+
+# Paths {#sec-path}
```

Comment:
Some more background motivation on why projects and paths are useful would be good. Things like project switching, collaboration, work portability, etc.

Reply:
I have added more background in the RStudio section.